mifosng-provider/src/main/java/org/mifosplatform/portfolio/loanaccount/rescheduleloan/service/LoanRescheduleRequestReadPlatformServiceImpl.java

```diff
@@ -145,7 +149,7 @@ public LoanRescheduleRequestData mapRow(final ResultSet rs, final int rowNum) th
 
 			return LoanRescheduleRequestData.instance(id, loanId, statusEnum, rescheduleFromInstallment, graceOnPrincipal, 
 					graceOnInterest, rescheduleFromDate, adjustedDueDate, extraTerms, interestRate, rescheduleReasonCodeValue, 
-					rescheduleReasonComment, timeline, clientName, loanAccountNumber, clientId, recalculateInterest);
+					rescheduleReasonComment, timeline, clientName, loanAccountNumber, clientId, recalculateInterest, null);
```

Comment:
First declare rescheduleReasons as null and then pass this variable so that readability will increase